src/mtx/index.ts

```diff
@@ -1,2 +1,3 @@
 export * as commands from './commands/index.js';
 export * as message from './message/index.js';
+export {default as CommandBinaryBuffer} from './utils/CommandBinaryBuffer.js';
```

Comment:
in utils should be index.js and here all utils should be exported

and constants

it's in my next mr

Reply:
code removed